frontend/src/pages/index.js

```diff
@@ -1,16 +1,32 @@
 import * as React from "react"
+import { Link } from "gatsby"
+import { StaticImage } from "gatsby-plugin-image"
+import homeImage from "../images/image245.png"
+import {ButtonLink} from "../components/Button";
 
 import Layout from "../components/layout"
 import Seo from "../components/seo"
 
 const IndexPage = () => (
   <Layout>
-      <div>Transform lives, transform the world section</div>
-      <div>Join us this sunday section</div>
-      <div>Welcome section</div>
-      <div>Get connected section</div>
-      <div>Get involved section</div>
-      <div>Upcoming events section</div>
+    <div className="w-full">
+      <div className="relative text-center text-Shades-0">
+        <img className="w-full h-full mb-0" src={homeImage}/>
+        <h1 className="absolute mb-0 top-1/2 -translate-y-1/2 left-1/2 -translate-x-1/2 line-clamp-2 xl:text-[60px] lg:text-[50px] md:text-[40px] sm:text-[35px] w-full text-[25px] font-bold leading-tight">Transform lives, <br/> Transform the world.</h1>
+      </div>
+      <div className="w-full bg-Neutral-200 justify-center flex px-4">
+          <div className="flex xl:flex-row lg:flex-row md:flex-row sm:flex-col flex-col justify-between items-center py-[50px] max-w-container w-full gap-4">
```

Comment:
```suggestion
          <div className="flex md:flex-row flex-col justify-between items-center py-[50px] max-w-container w-full gap-4">
```
have the same tailwind tags for several breakpoints is redundant. pick the minimum breakpoint and it should stick.   like i suggested.

Reply:
Oh awesome, didn't know that, will make these changes!